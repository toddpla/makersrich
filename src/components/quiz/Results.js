import React from 'react'

export default class Results extends React.Component {
  render() {
    return (
      <div>
        4/10 ur shite
      </div>
    )
  }
}
