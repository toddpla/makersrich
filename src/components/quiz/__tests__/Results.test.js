import React from 'react';
import { shallow } from 'enzyme'
import Results from '../Results'

let wrapper, results

beforeEach(function() {
  results = [true, false, false, true, true]
  wrapper = shallow(
    <Results results={results}/>
  )
});

test('it renders Results', () => {
  expect(wrapper).toMatchSnapshot()
})
