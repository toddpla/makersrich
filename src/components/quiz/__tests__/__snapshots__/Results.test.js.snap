// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Results 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Results
    results={
      Array [
        true,
        false,
        false,
        true,
        true,
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        3,
        " / 5 mate",
      ],
    },
    "ref": null,
    "rendered": Array [
      3,
      " / 5 mate",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          3,
          " / 5 mate",
        ],
      },
      "ref": null,
      "rendered": Array [
        3,
        " / 5 mate",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
